worldengine/generation.py

```diff
@@ -188,25 +182,25 @@ def generate_world(w, step):
         return w
 
     # Precipitation with thresholds
-    PrecipitationSimulation().execute(w, seed)
+    PrecipitationSimulation().execute(w, seed * 13)
```

Comment:
I wanted to centralize the seed-modifications all in one place so it is immediately visible that a change happens and everything could be easily changed later.